src/main/java/org/veupathdb/service/sequence/controller/SequenceController.java

```diff
@@ -0,0 +1,39 @@
+package org.veupathdb.service.sequence.controller;
+
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.Request;
+import jakarta.ws.rs.core.StreamingOutput;
+
+import org.veupathdb.lib.container.jaxrs.server.annotations.Authenticated;
+import org.veupathdb.service.sequence.generated.model.SequencePostRequest;
+import org.veupathdb.service.sequence.generated.resources.SequenceSequenceType;
+
+import org.veupathdb.service.sequence.util.StreamSequences;
+import org.veupathdb.service.sequence.util.Validate;
+import org.veupathdb.service.sequence.service.Sequences;
+
+public class SequenceController implements SequenceSequenceType {
+
+  @Context
+  private Request req;
```

Comment:
SOP here is to always inject org.glassfish.jersey.server.ContainerRequest into services, which Jersey uses to attach all request metadata encapsulated by the following jaxrs interfaces:

ContainerRequestContext, Request, HttpHeaders, PropertiesDelegate, PropertiesResolver